ArchUnitNET/Domain/PlantUml/Export/PlantUmlFileBuilder.cs

```diff
@@ -130,10 +134,15 @@ public PlantUmlFileBuilder WithDependenciesFrom(IEnumerable<Slice> slices, Gener
                 }
                 else if (!sliceIsPackage)
                 {
-                    nodes.Add(slice, new PlantUmlSlice(slice.Description, slice.NameSpace));
+                    var slc = new PlantUmlSlice(slice.Description, slice.NameSpace);
+                    if (generationOptions.C4Style)
+                    {
+                        slc.ChangeView();
```

Comment:
It doesn't sound like we want to switch here, but call `slc.UseC4Style()`, which internally sets `C4Style = true`